models/tests/a2j-template-test.js

```diff
@@ -22,7 +22,7 @@ describe('A2JTemplate Model', function () {
     let promise = A2JTemplate.findAll({ guideId: 1261 })
 
     return promise.then(function (a2jTemplates) {
-      assert.equal(a2jTemplates.length, 3, 'should have 3 total templates')
+      assert.equal(a2jTemplates.length, 4, 'should have 3 total templates')
```

Comment:
missed updating the 'should' message to 4

Reply:
Good catches, thank you!